dapps/marketplace/src/pages/ReferralRedirect.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react'
+import get from 'lodash/get'
+
+import { withRouter } from 'react-router-dom'
+
+import UserActivationLink from 'components/UserActivationLink'
+
+const localStorageKey = 'growth_invite_code'
+
+const ReferralRedirect = ({ match }) => {
+  const inviteCode = get(match, 'params.inviteCode')
+  const [redirect, setRedirect] = useState(false)
+
+  useEffect(() => {
+    if (inviteCode && inviteCode.length === 11) {
```

Comment:
This isn't accurate.  It appears invite code can be 7-11 characters, according to our prod DB.

CC @franckc

Reply:
Yeah, it's kinda funky but looks like the invite code can be between 7 to 11 characters!
Here is the code that generates those invite codes: https://github.com/OriginProtocol/origin/blob/master/infra/growth/src/resources/authentication.js#L204

Reply:
Thanks @mikeshultz. Will update it.